doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-subprocesses-ref.adoc

```diff
@@ -0,0 +1,74 @@
+[id='bpmn-subprocesses-ref_{context}']
+
+= BPMN2 subprocesses in process designer
+
+A subprocess is an activity that contains nodes. You can embed part of the main process within a subprocess. You can also include variable definitions within the subprocess. These variables are accessible to all nodes inside the subprocess.
+
+A subprocess must have one incoming connection and one outgoing connection. A terminate end event inside a subprocess ends the subprocess instance but does not automatically end the parent process instance. A subprocess ends when there are no more active elements in it.
+
+The following subprocess types are supported in {PRODUCT}:
+
+* Embedded subprocess, which is a part of the parent process execution and shares its data
+* Ad hoc subprocess, which has no strict element execution order
+* Reusable subprocess, which is independent from its parent process
+* Event subprocess, which is only triggered on a start event or a timer
+* Multi-instance subprocess
+
+In the following example, the Place Order subprocess checks whether sufficient stock is available to place the order and updates the stock information if the order can be placed. The customer is then notified through the main process based on whether or not the order was placed.
+
+image:BPMN2/subprocess.png[]
+
+.Embedded subprocess
+
+An embedded subprocess encapsulates a part of the process. It must contain a start event and at least one end event. Note that the element enables you to define local subprocess variables that are accessible to all elements inside this container.
+
+[[_adhoc_sub_process]]
+.AdHoc subprocess
+
+An ad hoc subprocess or process contains a number of embedded inner activities and is intended to be executed with a more flexible ordering compared to the typical routing of processes. Unlike regular processes, an ad hoc subprocess does not contain a complete, structured BPMN2 diagram description, for example, from start event to end event. Instead, the ad hoc subprocess contains only activities, sequence flows, gateways, and intermediate events. An ad hoc subprocess can also contain data objects and data associations. The activities within the ad hoc subprocesses are not required to have incoming and outgoing sequence flows. However, you can specify sequence flows between some of the contained activities. When used, sequence flows provide the same ordering constraints as in a regular process. To have any meaning, intermediate events must have outgoing sequence flows and they can be triggered multiple times while the ad hoc subprocess is active.
+
+.Reusable subprocess
+Reusable subprocesses appear collapsed within the parent process. To configure a reusable subprocess, select the reusable subprocess, click image:getting-started/diagram_properties.png[], and expand *Implementation/Execution*. Set the following properties:
```

Comment:
Question - Can you describe the icon rather than use an image? IBMSG 210p

Reply:
@csherrar We can, however, to be consistent with other sections, which also has icons, I think we can use icon here. 

wdyt?